generate.py

```diff
@@ -262,6 +265,9 @@ def check_positive(x):
                              "model output)",
                         type=float)
     parser.add_argument("-tm", "--tempo", help="approximate tempo of generated sample in BMP", type=check_positive)
+    parser.add_argument("-f", "--midi_input_file", help="Start the muisic off with a midi file", type=str)
```

Comment:
```suggestion
    parser.add_argument("-i", "--midi-prompt", help="continue a given midi input file", type=str)
```

Reply:
This suggestion and the one below are for consistency in style with how the other parser arguments have been written, and also a bit more specificity.

Reply:
Done